src/inmanta/server/agentmanager.py

```diff
@@ -404,7 +404,7 @@ async def _log_session_creation_to_db(
         Note: This method call is allowed to fail when the database connection is lost.
         """
         await data.AgentProcess.seen(tid, session.nodename, session.id, now)
-        await data.AgentInstance.log_instance_creation(tid, session.id, endpoint_names, now)
+        await data.AgentInstance.log_instance_creation(tid, session.id, endpoint_names)
```

Comment:
the now argument is important, these requests are processed asynchronously, not passing now makes the log lose temporal consistency iirc.

Reply:
Could you elaborate on this? I don't actually see any logging happening. The `now` parameter is not used by the `log_instance_creation` method.

Reply:
oh, ok, I assumed we have a creation time as well.